spec/system/admin/admin_creates_new_tag_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe "Admin creates new tag", type: :system do
+  let(:admin) { create(:user, :super_admin) }
+
+  before do
+    sign_in admin
+    visit new_admin_tag_path
+  end
+
+  def create_and_publish_tag
+    fill_in("Name", with: "Tag Name")
+    check "Supported"
+    fill_in("Short summary", with: "This is a tag")
+    click_button("Create Tag")
+  end
+
+  it "creates a new tag", :aggregate_failures do
+    expect(page).to have_content("New Tag")
+    create_and_publish_tag
+    expect(page.driver.status_code).to eq(200)
```

Comment:
System tests are meant to reflect the user level and I think that you had to go through `driver` to get to `status_code` hints at the fact that this isn't a common assertion in this type of test. I think we should assert on something that's less of a technical detail, e.g. the flash message:

```ruby
  def create_and_publish_tag(tag_name)
    fill_in("Name", with:tag_name)
    ...
  end

  it "creates a new tag", :aggregate_failures do
    ...
    create_and_publish_tag("Tag1")
    expect(page).to have_text("Tag1 has been created!")
  end
```

Reply:
I've made this change in a separate, follow-up PR!